lib/player/player-service.ts

```diff
@@ -13,49 +13,74 @@ export namespace playerService {
     minStartingBid: number;
     averageBuyNow: number;
     averageStartingBid: number;
-    samplecount: number;
+    sampleCount: number;
   }
+
   export async function getMarketPrice(
     resourceId: number
   ): Promise<MarketPrice> {
+    let invalidSamples = 0;
     const auctions = await getAuctions(resourceId);
-
     const price: MarketPrice = {
       minBuyNow: Number.MAX_VALUE,
       minStartingBid: Number.MAX_VALUE,
       averageBuyNow: 0,
       averageStartingBid: 0,
-      samplecount: 0
+      sampleCount: 0
     };
 
     auctions.forEach(a => {
-      if (a.itemData.resourceId != resourceId) return;
+      if (a.itemData.resourceId != resourceId) {

```

Comment:
I don't remember why I put this check here. Did you fall into this case while testing?

Reply:
No, i was just adding braces ;)